.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ name: CI
 # events but only for the master branch
 on:
   push:
-    branches: [ main ]
+    branches: [ main, merlin-domains ]
```

Comment:
And on pull_request isn't it ?

Reply:
My bad, done on [74f6f2c](https://github.com/ocaml/merlin/pull/1907/commits/74f6f2c1550b69b5ba23cfdaa819a2fecef1b251)